projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -42,12 +47,18 @@ export class TitledContentComponent {
   @Input()
   public titlePosition: TitlePosition = TitlePosition.Header;
 
+  @Input()
+  public titleStyle: TitledContentTitleStyle = TitledContentTitleStyle.Regular;
+
   @Input()
   public hideTitle: boolean = false;
 
   @Input()
   public link?: string;
 
+  @Input()
+  public headerPosition: TitledContentHeaderJustify = TitledContentHeaderJustify.Center;
```

Comment:
can we call it `headerJustifyContent: TitledContentHeaderJustify`

Reply:
fixed